viewer/packages/cad-model/src/utilities/getShaderMaterial.ts

```diff
@@ -7,7 +7,10 @@ import { RevealGeometryCollectionType } from '@reveal/sector-parser';
 import { assertNever } from '@reveal/utilities';
 import { RawShaderMaterial } from 'three';
 
+import assert from 'assert';
+
 export function getShaderMaterial(type: RevealGeometryCollectionType, materials: Materials): RawShaderMaterial {
+  assert(type !== RevealGeometryCollectionType.TexturedTriangleMesh);
```

Comment:
Well, logically we can't get the textured mesh material from the `Materials` object just by type, since every textured mesh sector has its own material. Also, this is only used by `createBatchBuffer`, which is used for instanced meshes and primitives, so not relevant